src/wp-admin/about.php

```diff
@@ -14,14 +14,14 @@
 /* translators: Page title of the About ClassicPress page in the admin. */
 $title = _x( 'About', 'page title' );
 
-list( $display_version ) = explode( '-', get_bloginfo( 'version' ) );
+$display_version = get_bloginfo( 'classicpress_display_version' );
 
 include( ABSPATH . 'wp-admin/admin-header.php' );
 ?>
 	<div class="wrap about-wrap full-width-layout">
 		<h1><?php printf( __( 'Welcome to ClassicPress&nbsp;%s' ), $display_version ); ?></h1>
 
-		<p class="about-text"><?php printf( __( 'Thank you for updating to the latest version! ClassicPress %s will smooth your design workflow and keep you safe from coding errors.' ), $display_version ); ?></p>
+		<p class="about-text"><?php printf( __( 'Thank you for trying ClassicPress! We are under heavy development while we prepare for an initial release.' ), $display_version ); ?></p>
```

Comment:
We should assume all strings are constant and future proof them where we can. I'd advised against saying the software is 'coming soon' because anyone using the software now will assume it's ready. It sends the wrong message.

Reply:
I don't necessarily agree here, this string reflects the reality of our current state. For me the moment to change it and think about permanence would be when we are closer to release, have translation infrastructure set up, etc.

Anyway this string has served its main purpose (generating the screenshot in this PR). What do you think it should be for now?

Reply:
can we close this?

Reply:
I've added issue #41 to track this as a follow-up task.